docs/content/configuration/expanding_pvc.md

```diff
@@ -2,9 +2,9 @@
 
 Use this information to expand existing volumes.
 
-**Important:** Before expanding an existing volume, be sure that the relevant StorageClass `.yaml` `allowVolumeExpansion` parameter is set to true. For more information, see [Creating a StorageClass](csi_ug_config_create_storageclasses.md).
+**Important:** Before expanding an existing volume, be sure that the relevant StorageClass `.yaml` `allowVolumeExpansion` parameter is set to true. For more information, see [Creating a StorageClass](creating_volumestorageclass.md).
```

Comment:
```suggestion
**Important:** Before expanding an existing volume, be sure that the relevant StorageClass YAML `allowVolumeExpansion` parameter is set to true. For more information, see [Creating a StorageClass](creating_volumestorageclass.md).
```
unless there's a reason to use `.yaml` instead of YAML in some (but not all?) places

Reply:
IMO we could also remove said "yaml" part in this case

Reply:
DONE